### DISCUSSION TITLE
[iSSUE]  Can this script install rtorrent with custom location?

### DISCUSSION THREAD

**Discussion by Suitear** [swizzin/swizzin#1132] [Q&A + Help]
### What happened?

hey, I'm on Debian 10 and Ubuntu 20.04, usingg rtinst for years, but recently the new rtinst script can't work on these platform so I'm looking into th swizzin. but I have tons of torrents like 20000+, so it would be hard to move to new platform. I am wondering if I can re-install rtorrent via Swizzin by using the existing configuration.
my rtorrent is located in /home/user/rtorrent/(download/watch, etc,)
my .rtorrent.rc file is in /home/user/
my rutorrent is located in /var/www/rutorrent/

So before turning to Swizzin, can it be customized to my current configuration? Thanks.

### Swizzin commit

nohashyet

### What OS are you using?

Ubuntu 20.04 (Focal)

### What architecture is your OS?

amd64

### Relevant logs and output

```plaintext
no logs.
```

**Comment by brettpetch** [comment]
Hi, 

I'd recommend first backing up both your sessions file and .rtorrent.rc before swapping to swizzin.

From there, `box install rtorrent rutorrent` it, stop the services, merge your configurations, and do startup. This should work seamlessly, but you'll also have a backup of the configuration if things 'hit the fan'. Swizzin should not delete files during setup in any case. rutorrent will install the latest version and will be located at /srv/ -- we also use our own flavour of nginx, so there may be some reconfiguration required on that end.

/b

**Comment by Suitear** [comment]
Thanks, but that's the problem. My torrent session is located in /home/user/torrent/. session/, how to change the files under it to adjust to the new location /src.? There are 20000+ files in this folder, with specific location info in every file.
If. Swizzin can customize Installation location, that would be perfect or support importing configuration from any existing rtorrent.
